src/applications/resources-and-support/components/SearchResultList.jsx

```diff
@@ -16,51 +16,58 @@ const articleTypes = {
   [ENTITY_BUNDLES.STEP_BY_STEP]: 'Step by step',
 };
 
+function SearchResult({ article }) {
```

Comment:
Should we move this into its own file you think? Up to you 🙂

Reply:
Yeah I suppose we should